pandas/io/pytables.py

```diff
@@ -1627,7 +1627,7 @@ def infer(self, handler):
         new_self.read_metadata(handler)
         return new_self
 
-    def convert(self, values, nan_rep, encoding, errors):
+    def convert(self, values, nan_rep, encoding, errors, **kwargs):
         """ set the values from this selection: take = take ownership """
```

Comment:
can you add a doc-string here

Reply:
types are a bonus

Reply:
I would just add ``start=None`` and ``stop=None`` instead of kwargs

Reply:
> I would just add start=None and stop=None instead of kwargs

done, commit upcoming

> can you add a doc-string here

I would love to add one! I have difficulties understanding the general purpose of the `convert()` method though and documenting `values`, `nan_rep`, `encoding`, and `errors` is pretty challenging given my lack of understanding. As far as I see the `start` and `stop` arguments are not really meaningful arguments here (in `IndexCol.convert()`), so I am also a bit helpless with documenting them! :-) How would you document `start` and `stop` here?